cmake/CMakeLists.txt

```diff
@@ -600,6 +603,14 @@ if (onnxruntime_EXTENDED_MINIMAL_BUILD AND NOT onnxruntime_MINIMAL_BUILD)
   set(onnxruntime_MINIMAL_BUILD ON)
 endif()
 
+if (onnxruntime_MINIMAL_BUILD)
```

Comment:
want to do the same check for onnxruntime_USE_WINML ?
(rather than disabling via build.py)